src/Files.App/Utils/Storage/Operations/FileOperationsHelpers.cs

```diff
@@ -580,9 +580,10 @@ public static Task<(bool, ShellOperationResult)> CopyItemAsync(string[] fileToCo
 					{
 						using ShellItem shi = new(fileToCopyPath[i]);
 						using ShellFolder shd = new(Path.GetDirectoryName(copyDestination[i]));
-
+						string fileName = GetIncrementalName(overwriteOnCopy, copyDestination[i], fileToCopyPath[i]);
```

Comment:
```suggestion
						var fileName = GetIncrementalName(overwriteOnCopy, copyDestination[i], fileToCopyPath[i]);

```
Use `var` whenever possible.